README.md

```diff
@@ -11,58 +11,46 @@ If you are interested in this work please be aware it is a relatively new initia
 ## OME Ansible repository
 
 
-This playbook uses Ansible 2.0 features (which is still in development), do not use `pip install ansible`.
-See http://docs.ansible.com/ansible/intro_installation.html for full installation instructions.
+This playbook uses Ansible 2.0 features. See
+[the Ansible documentation](http://docs.ansible.com/ansible/intro_installation.html)
+for full installation instructions.
 
 ### Installation
 
-- Create a virtualenv:
+- Create a virtual environment and install Ansible:
 
-        virtualenv venvs/ansible
+        virtualenv ~/venvs/ansible
+        ~/venvs/ansible/bin/pip install ansible
 
-- Clone ansible and submodules:
+- If using Ansible with OpenStack, you might want to install `shade`:
 
-        git clone git://github.com/ansible/ansible.git --recursive
-
-- Install dependencies listed in `setup.py`:
-
-        venvs/ansible/bin/pip install paramiko jinja2 PyYAML pycrypto six
+        ~/venvs/ansible/bin/pip install shade
 
 - Clone this repository:
 
         git clone https://github.com/openmicroscopy/infrastructure.git
 
--  Clone the repository containing the inventory, host and group vars files.
-   Ansible will automatically look for `host_vars` and `group_vars` directories in the parent directory of the inventory file.
+-   Clone the repository containing the inventory, host and group vars files.
+    Ansible will automatically look for `host_vars` and `group_vars`
+    directories in the parent directory of the inventory file.
 
 
 ### Running the development ansible
```

Comment:
How do you feel about removing this whole section since everyone should know how to use a virtualenv? If you want to keep it then change the title above, and delete `Alternatively adjust the paths in this script as necessary` and the script?